_docs_integrate/01-integrate-scenarios.md

```diff
@@ -15,35 +15,39 @@ Here you'll find everything you need to seamlessly dive into the world of enmesh
 
 ## Getting Started
 
-Begin your integration journey by familiarizing yourself with the Connector REST API and exploring the fundamental steps for integration.
+Begin your integration journey by familiarizing yourself with the [Connector REST API]({% link _docs_integrate/access-the-connector.md %}) and exploring the fundamental steps for integration. Gain insights into effectively utilizing the [Connector SDKs]({% link _docs_integrate/access-the-connector.md %}#accessing-the-connector-by-software-development-kits-sdk) to streamline and optimize your integration. Discover how events work and how you can leverage them in your application in the [event introduction]({% link _docs_integrate/event-introduction.md %}).
 
-## Data Model
+## Get in touch with other Identities
+
+Learn how to [establish a Relationship to another Identity]({% link _docs_integrate/establish-a-relationship-to-another-identity.md %}).
 
-Understand enmeshed's data model and how it fits into your integration, including error codes and attribute values.
+## Working with Requests
 
-## Handling Events
+Learn how to create and manage Requests. Check out the [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}) and the [Requests over Templates]({% link _docs_integrate/requests-over-templates.md %}) and [Requests over Messages]({% link _docs_integrate/requests-over-messages.md %}) guides.
 
-Discover how events work and how you can leverage them in your application.
+## Manage Attributes of yourself
 
-## Manage Requests
+Explore how to [create Attributes for yourself]({% link _docs_integrate/create-attribute-for-yourself.md %}) and how to [share Attributes with a peer]({% link _docs_integrate/share-own-attribute-to-peer.md %}). Furthermore, learn how to [succeed Attributes to update their values]({% link _docs_integrate/succeeding-attributes-to-update-their-values.md %}).
 
-Learn how to create and manage requests.
+## Manage Attributes of others
 
-## Manage Attributes of Others
+Explore how to [query Attributes]({% link _docs_integrate/read-attribute-from-peer.md %}) and [store Attributes]({% link _docs_integrate/create-attribute-for-peer.md %}) in the Wallet for others.
 
-Explore how to store and query attributes in the Wallet for other users.
+## Basic communication
 
-## Messages
+Explore how to [send Messages]({% link _docs_integrate/sending-messages.md %}) through enmeshed for simple and secure communication.
```

Comment:
Sending Messages "through" sounds kind of odd (I know you just copied it). Maybe rather reformulate it using "via".